doc/rtd/explanation/format.rst

```diff
@@ -223,4 +324,5 @@ appliances. Setting ``allow_userdata: false`` in the configuration will disable
 
 .. _make-mime: https://github.com/canonical/cloud-init/blob/main/cloudinit/cmd/devel/make_mime.py
 .. [#] See your cloud provider for applicable user-data size limitations...
+.. _hook: https://en.wikipedia.org/wiki/Hooking
```

Comment:
oops, sorry I meant within cloud-init do we define the other hooks? We say: "This is the earliest `hook`_ to run, it runs during the `Network boot stage`.", which makes me wonder what the other hooks are and if there is a place in the documentation that I can see the order of these hooks. In this context does "hook" mean "handler"? Confusingly, on the [boot order](https://cloudinit.readthedocs.io/en/latest/explanation/boot.html#network) page there is a statement "A part-handler and [boothooks](https://cloudinit.readthedocs.io/en/latest/explanation/format.html#cloud-boothook) will run at this stage.". Based on my understanding, a boothook _is_ one of the part handlers, but maybe I'm wrong on that.

Reply:
oh yeah, I have struggeled to get other hooks in cloud-init. Interestingly, in the code, the boothook in under handlers.